controller/api_test.go

```diff
@@ -214,11 +215,7 @@ func TestEnvironmentApi(t *testing.T) {
 
 	e, err := g.client.EnvironmentGet(m)
 	panicIf(err)
-	assert(testEnv.Spec.AllowedFunctionsPerContainer == e.Spec.AllowedFunctionsPerContainer, "env AllowedFunctionsPerContainer should match after reading")
-	assert(testEnv.Spec.Poolsize == e.Spec.Poolsize, "env Poolsize should match after reading")
-	assert(testEnv.Spec.Builder == e.Spec.Builder, "env Builder should match after reading")
-	assert(testEnv.Spec.Runtime == e.Spec.Runtime, "env Runtime should match after reading")
-	assert(testEnv.Spec.Version == e.Spec.Version, "env Version should match after reading")
+	assert(reflect.DeepEqual(testEnv.Spec, e.Spec), "env should match after reading")
```

Comment:
Suggest using google/go-cmp as a safer alternative to `reflect.DeepEqual`, may be as a separate issue if you would like

Reply:
I didn't know about google/go-cmp. How is it different/safer from the reflect approach? Anyway, for the sake of avoiding too much lingering, let's defer this to a later pr.